Number_Conversions/Binary_To_Decimal/Binary_To_Decimal.java

```diff
@@ -0,0 +1,31 @@
+// Java program to convert binary to decimal 
+
+// Function to convert binary to decimal 
+class numberconversion { 
+	static int binaryToDecimal(int n) 
```

Comment:
How are you testing whether it is working or not?

Reply:
On  online ide its working